### PR TITLE
update docs

### PR DESCRIPTION
**Pull request by dwertent**

### CODE REVIEW

**Comment by craigbox** [comment]
Let's remove most of this, as it's moved into kubescape.io?

**Comment by dwertent** [comment]
@craigbox We should have a separate PR for moving the docs to kubescape.io

**Comment by craigbox** [comment]
I mean, most of them are there already

**Comment by Oshratn** [comment]
@matthyx asking for your approval, for another set of eyes.

File: README.md [hunk 1]
```diff
@@ -56,20 +68,21 @@ _Did you know you can use Kubescape in all these places?_
 ## Under the hood
 
 Kubescape uses [Open Policy Agent](https://github.com/open-policy-agent/opa) to verify Kubernetes objects against [a library of posture controls](https://github.com/kubescape/regolibrary).
+For images scanning, it uses [Grype](https://github.com/anchore/grype).
```
Comment (matthyx) [inline README.md:71]: wanna add copacetic here too?

File: README.md [hunk 2]
```diff
@@ -68,20 +78,22 @@ Kubescape can be used as a GitHub Action. This is a great way to integrate Kubes
...
```
Comment (matthyx) [inline README.md:82]:
```suggestion
For image patching, it uses [Copacetic](https://github.com/project-copacetic/copacetic).
```

**Review by matthyx — APPROVED** [review]

**Review by Oshratn — APPROVED** [review]

**Comment by Oshratn** [comment]
@matthyx please merge, no authorization on my side.

**Pull request merged**